plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -282,6 +283,9 @@ def run(self):
                     if not (skip or self.skip_repomd_steps):
                         purge.remove_repo_duplicate_nevra(self.conduit.repo_id)
 
+                if not self.metadata_found:
```

Comment:
An in-line comment here would be helpful just to give a hint that this will allow the mirror list case to try another URL.